dev/bots/analyze-sample-code.dart

```diff
@@ -868,8 +866,8 @@ class Section {
 /// analyzed.
 class Snippet {
   Snippet({this.start, List<String> input, List<String> args, this.serial}) {
-    this.input = <String>[]..addAll(input);
-    this.args = <String>[]..addAll(args);
+    this.input = <String>[...input];
+    this.args = <String>[...args];
```

Comment:
there's probably cleaner ways to do this than either of those, heh

Reply:
😬 I wish I knew of one!

Reply:
`this.input = input.toList()`?